src/views/planningPoker.tsx

```diff
@@ -93,20 +101,40 @@ const VoteList = ({ votes }) => (
 );
 
 const VoteAnalysis = ({ votes }) => {
-  const estimates = votes.map((v) => v.estimate);
-  const min = Math.min(...estimates);
-  const avg = estimates.reduce((n, sum) => n + sum, 0) / estimates.length;
-  const max = Math.max(...estimates);
 
-  return (
+  const estimates = []
+//  const estimates = votes.map((v) => v.estimate);
+  for (let i = 0; i < votes.length; i++) {
+    const est = votes[i].estimate;
+
+    if (!isNaN(Number(est))) {
+      estimates.push(Number(est));
+    }
+  }   
+    var min = "?"
+    var avg = "?"
+    var max = "?"
+
+    if (estimates.length !== 0) {
+    
+    min = Math.min.apply(Math, estimates);
+    max = Math.max.apply(Math, estimates);
+    var sum = 0;
+    estimates.forEach(function(num) { sum += num });
+    avg = sum / estimates.length;
```

Comment:
Could we keep our original calculation methods here instead?

```suggestion
    min = Math.min(...estimates);
    avg = estimates.reduce((n, sum) => n + sum, 0) / estimates.length;
    max = Math.max(...estimates);
```

Reply:
done